src/Styles/Blocks/Card.css

```
@@ -0,0 +1,93 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap");
+
+body {
```

```suggestion
html {
```
it's better to set the font to the `html` selector tag